src/pages/Explore/index.tsx

```diff
@@ -53,23 +55,25 @@ const FiltersWrapper = styled.div`
 
 const Explore = () => {
   return (
-    <ExploreContainer>
-      <TitleContainer>Explore Tokens</TitleContainer>
-      <FiltersWrapper>
-        <FiltersContainer>
-          <NetworkFilter />
-          <FavoriteButton />
-          <TimeSelector />
-        </FiltersContainer>
-        <SearchContainer>
-          <SearchBar />
-        </SearchContainer>
-      </FiltersWrapper>
+    <Trace page={PageName.POOL_PAGE} shouldLogImpression>
```

Comment:
should be Explore page (not Pool page)

Reply:
OH RIP, must have changed when I merged -- thanks good catch!